### ISSUE TITLE
Allow passing of extra configuration to Sx-Cat

### ISSUE THREAD

**Issue by pacesm** [EOX-A/ngeo-b#48]
The Sx-Cat harvester may require additional configuration options which need to be loaded before the harvesting and BR ingestion feed is started.

Ideally this should be done in the layer definition, e.g
```
<browseLayer browseLayerId="...">
    ...
    <harvestingSource>...</harvestingSource>
    <harvestingExtraConfiguration><![CDATA[
[harvesting]
record_filters =
    productType, EQ, OCM2_LA_1B
    beginAcquisition, RegEx, ^20(19|20)
]]></Configuration>
</browseLayer>
```

If the config cannot be included in the layer definition, an alternative could be an option to prevent the automatic harvest start after the layer insert allowing a later manual change of the Sx-Cat configuration.

**Comment by pacesm** [comment]
implemented in d9963fac4248396bdc354507c80844c40d11bdbe

It is now possible to specify multiple harvesting sources and pass the extra Sx-Cat configuration, e.g.,
```
<?xml version="1.0" encoding="UTF-8"?>
<synchronizeConfiguration xmlns="http://ngeo.eo.esa.int/schema/configurationElements">
    <startRevision>0</startRevision>
    <endRevision>2</endRevision>
    <removeConfiguration/>
    <addConfiguration>
        <browseLayers>
            <browseLayer browseLayerId="SAR-ASA_IMS_1P">
                <browseType>SAR-ASA_IMS_1P</browseType>
                <title>Envisat ASAR and ERS SAR IMS 1P</title>
                <description>Level 1 products for  Envisat ASAR Image Mode and ERS SAR Single-Look Complex</description>
                <browseAccessPolicy>OPEN</browseAccessPolicy>
                <hostingBrowseServerName/>
                <relatedDatasetIds>
                    <datasetId>ASA_APP_1P.data</datasetId>
                </relatedDatasetIds>
                <containsVerticalCurtains>false</containsVerticalCurtains>
                <shortenIngestedInterval>100.0</shortenIngestedInterval>
                <highestMapLevel>10</highestMapLevel>
                <lowestMapLevel>6</lowestMapLevel>
                <grid>urn:ogc:def:wkss:OGC:1.0:GoogleCRS84Quad</grid>
                <timeDimensionDefault>2012</timeDimensionDefault>
                <tileQueryLimit>100</tileQueryLimit>
                <strategy>skip</strategy>
                <harvestingSource name="SAR_IMS_1P">https://.../SAR_IMS_1P/index/</harvestingSource>
                <harvestingSource name="ASA_IMS_1P">https://.../ASA_IMS_1P/index/</harvestingSource>
                <harvestingConfiguration type="sxcat"><![CDATA[record_filters =
    beginAcquisition, RegEx, ^2005-
]]></harvestingConfiguration>
            </browseLayer>
        </browseLayers>
    </addConfiguration>
</synchronizeConfiguration>
```

The limitation is that the havesting configuration is not stored in layer DB model.

**Comment by lubojr** [comment]
Added in `branch-4-1`. Harvesting configs do not need to be stored in DB for now.
